includes/wikia/GlobalFunctions.php

```diff
@@ -1557,3 +1557,17 @@ function wfGetBaseDomainForHost( $host ) {
 
 	return $wgWikiaBaseDomain;
 }
+
+function wfDoesURLHaveFandomDomain( $url ): bool {
+	global $wgFandomBaseDomain;
+
+	// e.g. not existing wikis
+	if ( empty( $url ) ) {
+		return false;
+	}
+
+	$host = parse_url( $url, PHP_URL_HOST );
+	$host = WikiFactory::normalizeHost( $host );
```

Comment:
If https://github.com/Wikia/app/pull/16068 gets merged first, this should be updated as the `normalizeHost` method is going to be moved out of `WikiFactory` class by @themech